velox/type/Filter.h

```diff
@@ -1083,8 +1154,26 @@ class BytesValues final : public Filter {
     upper_ = *std::max_element(values_.begin(), values_.end());
   }
 
-  std::unique_ptr<Filter> clone() const final {
-    return std::make_unique<BytesValues>(*this);
+  BytesValues(const BytesValues& other, bool nullAllowed)
```

Comment:
Instead of repeating code (which will need changes twice if we change the logic), we can do something like re-using the above constructor/assigning the value directly. Its also more efficient imho (I guess)?
 
https://github.com/facebookincubator/velox/pull/297/commits/a12933b7815fa147e351f64e8bf866bd2f4f0761#diff-280c77e7a8b1d7cfaeaaed70f8ac11c67e881b4f1682ee099bc326a1b53e9349

Reply:
I actually tried this style, for example:
 BytesValues(const BytesValues& other, bool nullAllowed) {
    // call the existing constructor.
}

I am from Java world and this is how usually Java works.

However the compile complains and say bt this way the default base class is not initialized. Do you know how to fix this issue?

Reply:
Not sure about the compiler complaining. When I did the same as pointed in the diff, it seemed to work.